semgrep-core/src/engine/Eval_generic.mli

```diff
@@ -35,5 +35,8 @@ val parse_json : Common.filename -> env * code
 (* for MetavarCond *)
 val bindings_to_env : Config_semgrep.t -> Metavariable.bindings -> env
 
+(* For entropy analysis and other string analyzers *)
```

Comment:
I would maybe add that the mvar parameter is used just for debugging/logging purpose.